indigo/distribution.yaml

```diff
@@ -8711,7 +8711,7 @@ repositories:
       url: https://github.com/So-Cool/report_card.git
       version: master
     status: maintained
-  repository_name:
+  hansagv_description:
```

Comment:
This is the right name. When there is only one package with the same name as the repository at the root, you should remove the list of packages as the above conditions are assumed if there is no other information. And if it's assumed we don't want to restate the information. So you should use this version but remove lines 8716 and 8717